pkg/pillar/cmd/zedrouter/networkinstance.go

```diff
@@ -870,14 +863,14 @@ func recordIPAssignment(ctx *zedrouterContext,
 }
 
 // Add to an IPv4 address
-func addToIP(ip net.IP, addition uint) net.IP {
+func addToIP(ip net.IP, appNumOnUNet int) net.IP {
```

Comment:
It doesn't make sense to change the name and type for this function. It still adds N to the IP address. The caller merely uses it differently.

Reply:
Sure, done.